### PR TITLE
Removed unnecessary files from coveralls.

### LINKED ISSUE #568: Remove unnecessary files from coveralls.
Not all files need to be covered on coveralls. Files not required in coveralls are: 

- [ ] accounts/accountUtils.go
- [ ] cmd/root.go
- [ ] cmd/struct-utils.go
- [ ] path/mocks
- [ ] utils/struct-utils.go

### PR DESCRIPTION
**Pull request by Shrikant1212**
# Description
Configured CircleCI to update coverage report

Fixes #568

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -17,10 +17,19 @@ jobs:
           name: "Executing test cases"
           command: |
             docker run --rm -v $(pwd):/test --name go  razor-test go-acc ./... --ignore razor/accounts/mocks --ignore razor/cmd/mocks --ignore razor/utils/mocks --ignore pkg --output /test/coverage.txt
+            docker run --rm -v $(pwd):/test --name go  razor-test cat /test/coverage.txt > coverage.txt
+
+      - run:
+          name: "Removing unnecessary files from coveralls report"
+          command: |
+            sed -i '/razor\/accounts\/accountUtils.go/d;/razor\/cmd\/root.go/d;/razor\/cmd\/struct-utils.go/d;/razor\/path\/mocks*/d;/razor\/utils\/struct-utils.go/d' coverage.txt
```
Comment (ashish10677) [inline .circleci/config.yml:25]: Can we keep all the unnecessary files as a list in another file(json/text) and then ignore all those present in that file?

Comment (ashish10677) [reply .circleci/config.yml:25]: @SkandaBhat any thoughts on this?

File: .circleci/config.yml [hunk 2]
```diff
@@ -17,10 +17,19 @@ jobs:
...
       - run:
           name: "Publish Coverage to Coveralls.io"
           command: |
-            docker run --rm -v $(pwd):/test --name go -e COVERALLS_TOKEN=$COVERALLS_TOKEN razor-test goveralls -coverprofile=/test/coverage.txt -service semaphore -repotoken $COVERALLS_TOKEN
+            go get github.com/mattn/goveralls
+            goveralls -coverprofile=coverage.txt -service semaphore -repotoken $COVERALLS_TOKEN
+          # docker run --rm -v $(pwd)/coverage.txt:/report/coverage.txt --name razor-go -e COVERALLS_TOKEN=$COVERALLS_TOKEN razor-test cat /report/coverage.txt
```
Comment (ashish10677) [inline .circleci/config.yml:31]: You can remove this if it is not required.

**Review by ashish10677 — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (Shrikant1212) [reply .circleci/config.yml]: Done

**Comment by Shrikant1212** [comment]
@ashish10677  can help in reviewing https://coveralls.io/jobs/94472233

**Comment by SkandaBhat** [comment]
@Shrikant1212 we cannot merge unverified commits. Please create a new PR with verified commits.

**Pull request closed without merging**